testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/assertions/DecisionSelectors.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.process.test.api.assertions;
+
+import io.camunda.client.api.search.filter.DecisionInstanceFilter;
+import io.camunda.client.api.search.response.DecisionInstance;
+
+/** A collection of predefined {@link DecisionSelector}s. */
+public class DecisionSelectors {
+
+  /**
+   * Select the decision instance by its name.
+   *
+   * @param decisionName the definition name of the decision.
+   * @return the selector
+   */
+  public static DecisionSelector byName(final String decisionName) {
+    return new DecisionNameSelector(decisionName);
+  }
+
+  /**
+   * Select the decision instance by its name.
+   *
+   * @param decisionName the definition name of the decision.
+   * @param processInstanceKey the associated process instance.
+   * @return the selector
+   */
+  public static DecisionSelector byName(final String decisionName, final long processInstanceKey) {
```

Comment:
:wrench: We could add another selector: `byProcessInstanceKey()`. 

Assuming that we evaluate a business rule task and want to assert the DMN evaluation, the process instance key should be enough without decision name or ID.